data/preset/news.yaml

```diff
@@ -42,7 +42,7 @@ defines:
     contents:
     - type: ContentGroup.DynamicTitle
       properties:
-        format-string: !translate "see more from %s"
+        format-string: !translate "More news from %s"
```

Comment:
okay, good point. I won't worry about that for this release, I'll change it back, but I'd like to at least capitalize the "S" in "see more"